app/uk/gov/hmrc/gform/gform/HtmlSanitiser.scala

```diff
@@ -43,7 +43,6 @@ object HtmlSanitiser {
     doc.select(".hmrc-banner").remove()
     doc.getElementsByTag("link").remove()
     doc.getElementsByTag("meta").remove()
-    doc.getElementsByTag("noscript").remove()
```

Comment:
we use <noscript/> to disable backlink now. Leaving it may have an impact, have you checked?

Reply:
Reverted back.